code/book/CommitOrder.js

```diff
@@ -1,10 +1,22 @@
 var dates = require('dates')
 var config = require('config')
-module.exports.function = function (order) {
+var console = require('console')
+module.exports.function = function (order, $vivContext) {
+  var relevantHours = getLocalizedRelevantHours($vivContext)
+  console.info($vivContext.locale, "relevant hours", relevantHours)
   return {
     $id:  "" + Math.floor(Math.random() * 10000000),
     item: order.item,
     buyer: order.buyer,
-    relevantDateTime: dates.ZonedDateTime.fromDate(order.item.dateInterval.start).minusHours(config.get("relevantHours")).getDateTime()
+    relevantDateTime: dates.ZonedDateTime.fromDate(order.item.dateInterval.start).minusHours(relevantHours).getDateTime()
   }
 }
+
+function getLocalizedRelevantHours($vivContext) {
+  switch($vivContext.locale.split('-')[0]) {
```

Comment:
Instead of splitting the `$can.locale`, we could use `$can.language` and it would directly give us the 2 letter language code